marklogic-data-hub-central/ui/e2e/cypress/integration/persistence/persistence.spec.tsx

```diff
@@ -48,6 +48,22 @@ describe("Validate persistence across Hub Central", () => {
     cy.get("[aria-label=\"icon: caret-up\"]").should("have.attr", "class").and("match", /hc-table_activeCaret/);
   });
 
+  it("Go to curate tile, and validate that the accordion and tabs are kept when switching between pages", () => {
+    toolbar.getCurateToolbarIcon().click();
+    loadPage.getAccordionButton(0).click();
```

Comment:
Looks fixed now. One more thing, I'm not sure if "loadPage" is the proper page to put these actions in. I think the one we want to use here may be "curatePage' just to be organized.